media/css/mozorg/home/home-promo.less

```diff
@@ -873,6 +873,18 @@ html[lang|="en"] {
     }
 }
 
+.promo-small-landscape.introducing-webmaker {
+    background: #0ad; // bright blue
+
+    h2 {
+        padding: 30px 20px 20px 140px;
+        min-height: 64px;
+        background: url(/media/img/home/voices/promos/webmaker/webmaker-logo.svg) 15px 15px no-repeat;
+        .background-size(100px, 100px);
+        .font-size(1.1rem);
```

Comment:
Nit: I know it works the same, but use `px` value here for consistency? (it get's converted to `rem` units anyway, and keeps for consistency with other promos/pages).